scripts/CEE/collect-verbose-ca-logs/script.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
```

Comment:
Suggest to add
```
set -e
set -o nounset
set -o pipefail
```
eg https://github.com/openshift/managed-scripts/blob/8d6d682428d559461ddd209ca993f86067bd99f5/scripts/CEE/delete-worker-node/script.sh#L3C1-L5C16

This arguments can safeguard the script in case any failures in the middle.

Reply:
I added these arguments and retested the script, it is working as expected. Adding the updated script file.